### DISCUSSION TITLE
Change of display of complex objects in 2024

### DISCUSSION THREAD

**Discussion by R4cOOn** [datalust/seq-tickets#2087] [Help] [answered]
Hi,

When showing events in pre-2024 version, complex objects appeared "collapsed":
![image](https://github.com/datalust/seq-tickets/assets/1311219/e9eacbda-1da5-4ec9-901e-cb3e2e6bf0e7)
Since version 2024.x the same object looks like:
![image](https://github.com/datalust/seq-tickets/assets/1311219/eb346384-1e53-4190-8d74-ce3008a267d1)

The new display way is taking too much screen space and makes reading events harder.  
Is there a way to get back the old display?

Cheers.

**Comment by nblumhardt** [comment, marked as answer]
Hi @R4cOOn thanks for the note.

Yes, this is (your user avatar) > Preferences > Flatten the event properties view:

![image](https://github.com/datalust/seq-tickets/assets/342712/295fac0c-d175-4af8-9f87-9602807ca26b)

Unchecking this will switch to the alternative, nested view.

Let me know if this helps,
Nick

**Reply by R4cOOn** [reply]
Thank you for the quick reply!

A note somewhere in the release notes would have been helpful 
(it may have been there, in which case I missed it).

**Reply by nblumhardt** [reply]
Thanks @R4cOOn; I raised #2091 and added it to the original release milestone to make this more discoverable 👍
